src/app/containers/Copyright/index.jsx

```diff
@@ -5,14 +5,14 @@ import Copyright from '@bbc/psammead-copyright';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
 const CopyrightContainer = ({ children }) => {
-  const { imageCopyrightOffscreenText } = useContext(ServiceContext);
+  const { imageCopyrightOffscreenText, lang } = useContext(ServiceContext);
 
   return (
     <Copyright>
       {imageCopyrightOffscreenText ? (
         <VisuallyHiddenText>{imageCopyrightOffscreenText}</VisuallyHiddenText>
       ) : null}
-      {children}
+      {lang === 'en-GB' ? children : <span lang="en-GB"> {children}</span>}
```

Comment:
As far as I can see, theres no snapshot that tests this functionality, can you add one please?

(a test that uses a service context where lang === `en-GB` to see the snapshot without the `lang` on the span)